packages/aws-client/src/DynamoDb.ts

```diff
@@ -137,6 +141,54 @@ export const scan = improveStackTrace(
   }
 );
 
+/**
+ * Do a parallel scan of DynamoDB table using a document client.
+ *
+ * See https://docs.aws.amazon.com/amazondynamodb/latest/developerguide/Scan.html#Scan.ParallelScan.
+ * See [DocumentClient.scan()](https://docs.aws.amazon.com/AWSJavaScriptSDK/latest/AWS/DynamoDB/DocumentClient.html#scan-property).
+ *
+ * @param {number} totalSegments
+ *   Total number of segments to divide table into for parallel scanning
+ * @param {DocumentClient.ScanInput} scanParams - Params for the DynamoDB client scan operation
```

Comment:
Nit: Might be helpful to add https://docs.aws.amazon.com/amazondynamodb/latest/APIReference/API_Scan.html here, it always takes 30s to look it up each time

Reply:
e24e614dbb460405d887eaac9e0f4ce6a94cfd06